libraries/mbed/targets/hal/TARGET_STM/TARGET_NUCLEO_F103RB/objects.h

```diff
@@ -43,6 +43,7 @@ struct gpio_irq_s {
     IRQn_Type irq_n;
     uint32_t irq_index;
     uint32_t event;
+    PinName pin;
```

Comment:
You added the pin variable, but can't find in this PR where you assign a value to it. It's used in the lines 295, 277, but dont see any write to it.